src/scss/base.scss

```diff
@@ -93,6 +93,49 @@ body {
   margin: 0 0 15px 0;
 }
 
+.footer-container {
+  background: $orange;
+  height: 100%;
+  max-height: 300px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.footer-form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.footer-info {
+  font-size: 13pt;
+  letter-spacing: 1px;
+  color: white;
+  margin: 30px 0;
+  text-align: center;
+}
+
+.footer-text-field {
+  border-radius: 12px;
+  color: $dark-grey;
+  height: 40px;
+  width: 300px;
```

Comment:
use:
```
width: 100%;
max-width: 300px;
```
this will make the input shrink if it's container is not big enough while a fixed width in pixels might break your layout and you'll need to add media queries to fix it.